### PR TITLE
PC: dynamically set PYTHONPATH from real openpilot path

### PR DESCRIPTION
**Pull request by sshane**
As a person who installs openpilot at `~/openpilot/openpilot` I find myself changing the openpilot_env.sh file often to make sure my PYTHONPATH is correct for the openpilot modules. This should let you install openpilot wherever you want!

### CODE REVIEW

File: tools/openpilot_env.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 if [ -z "$OPENPILOT_ENV" ]; then
-  export PYTHONPATH="$HOME/openpilot:$PYTHONPATH"
+  OP_ROOT="$(dirname "$(dirname "${BASH_SOURCE[0]}")")"
```
Comment (pd0wm) [inline tools/openpilot_env.sh:2]:
```suggestion
  OP_ROOT="$( cd $(dirname "$(dirname "${BASH_SOURCE[0]}")") && pwd )"
```

Think you need this to get the absolute path.

Comment (sshane) [reply tools/openpilot_env.sh:2]: Both work for me on Ubuntu 20.04 opening a new terminal window, is this suggestion safer?

Comment (pd0wm) [reply tools/openpilot_env.sh:2]: Not sure how it works for you. For me the `OP_ROOT` is jus equal to `.`

Comment (sshane) [reply tools/openpilot_env.sh:2]: Oh if you print `OP_ROOT` when sourcing from your .bashrc it gets the absolute path, but when you source or run the script outside `.bashrc` it's relative.

**Pull request merged**